styles.css

```diff
@@ -1 +1,66 @@
-/*Maak hier jouw CSS!*/
```

Comment:
Volgens de uitwerking had hier een CSS reset van padding en margin moeten staan met een root selector. Ik vond dit ook niet duidelijk te zien in de opdracht, maar ik denk dat een reset een beetje standaard word gebruikt. Als je de uitwerking van Nova met je eigen versie in een browser vergelijkt zie je duidelijk waarom dit gebruikt word.